src/ManifestStrategy/GlossaryManifestStrategy.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace SprykerSdk\Integrator\ManifestStrategy;
+
+use SprykerSdk\Integrator\Dependency\Console\InputOutputInterface;
+
+class GlossaryManifestStrategy extends AbstractManifestStrategy
+{
+    /**
+     * @var string
+     */
+    protected const GLOSSARY_FILE_EXIST_ERROR = 'File `%s` does not exist. Please check file path or customize using `IntegratorConfig::getGlossaryFilePath()`.';
```

Comment:
As to me usage of this constant doesn't make any sense as they are not fully reusable.